### PR TITLE
move blank certificate page to dashboard

### PR DESCRIPTION
**Pull request by davidsbailey**
Finishes [PLAT-1536](https://codedotorg.atlassian.net/browse/PLAT-1536).

## Testing story

new unit test, new UI test

## Follow-up work

Actually updating the links which point to /certificates/blank will happen as part of the PR which launches the congrats pages on dashboard.

### CODE REVIEW

File: apps/src/sites/code.org/pages/public/certificates/blank.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import experiments from '@cdo/apps/util/experiments';
+
+const studioCertificate = experiments.isEnabled(experiments.STUDIO_CERTIFICATE);
+
+if (studioCertificate) {
```
Comment (dmcavoy) [inline apps/src/sites/code.org/pages/public/certificates/blank.js:5]: What will happen if someone hits this page without the experiment?

File: dashboard/config/routes.rb [hunk 2]
```diff
@@ -468,6 +468,7 @@
 
     get '/print_certificates/:encoded_params', to: 'print_certificates#show'
 
+    get '/certificates/blank', to: 'certificates#blank'
```
Comment (dmcavoy) [inline dashboard/config/routes.rb]: Do you need the to part here?

File: apps/src/sites/code.org/pages/public/certificates/blank.js (hunk 1 above)
Comment (davidsbailey) [reply apps/src/sites/code.org/pages/public/certificates/blank.js:5]: this js file won't do anything if someone hits this page without the experiment, and the user will get the original pegasus experience served by certificates.haml.

File: dashboard/config/routes.rb (hunk 2 above)
Comment (davidsbailey) [reply dashboard/config/routes.rb]: good catch! TIL

**Review by dmcavoy — APPROVED** [review]

**Pull request merged**